libs/core/src/lib/card/card.component.ts

```diff
@@ -83,6 +91,11 @@ export class CardComponent implements OnChanges, OnInit, CssClassBuilder, OnDest
         this.buildComponentCssClass();
     }
 
+    /** @hidden */
+    ngAfterContentInit(): void {
+        this.listItems?.first?.setIsFirst(true);
```

Comment:
the list can be added later (after content init), so it won't be caught. probably we have to manage the list item's tabindex in the list item code.

Reply:
`the list can be added later (after content init), so it won't be caught`

Confused what you mean here, the card won't necessarily have a list so we need the ? check

`probably we have to manage the list item's tabindex in the list item code.`

That is what's happening in the `setIsFirst` function

Reply:
He meant that ngAfterContentInit is triggered only once. What if the component did not have a list in it during content init, but after some conditions were met in the client's app and the list appeared? In that case, this `setIsFirst` will not get triggered